app/controllers/users_controller.rb

```diff
@@ -45,6 +46,42 @@ def index
     end
   end
 
+  def show
+    user = User.find(params.require(:id))
+
+    render status: :ok, json: user.as_json(
+      include: %w[roles],
+      methods: %w[special_account_competitions],
+      private_attributes: %w[email],
+    )
+  end
+
+  def merge
+    to_user = User.find(params.require(:toUserId))
+    from_user = User.find(params.require(:fromUserId))
```

Comment:
Nitpick: These lines should perhaps be swapped, so that you get a "from --> to" impression when reading as well.